README.md

```diff
@@ -27,26 +27,36 @@ or use the provided [requirements.txt](requirements.txt) file provided in this r
 ```
 pip install -r requirements.txt
 ```
-# Release Versioning Annotations  
-synapseAnnotations release version tags structure **v0.0.0** fits within [semantic versioning](http://semver.org/) guidelines. 
-The tags release numbering scheme is defined as: 
+
+## Proposing changes
+
+1. Make changes on your feature branch.
+1. Request and complete a review from someone on the team.
```

Comment:
Possibly allow a way to contact people on the team (in the case of external people)